fc/jira/tasks.py

```diff
@@ -1,124 +1,35 @@
-import datetime
-import re
 import requests
-from fc.jira.backlog_task import BacklogTask
 from requests import HTTPError
 from requests.auth import HTTPBasicAuth
 
 from fc.exceptions.task_exception import TaskException
-# from .triage_task import TriageTask
-from . import triage_task
 from ..auth.auth import Auth
-from .task import Task
-from typing import Tuple, Optional
 
 
 search_url = 'https://jira.cms.gov/rest/api/2/search?jql='
+api_url = 'https://jira.cms.gov/rest/api/2/issue/'
 
-importance_dict = {
-    'High': 10,
-    'high': 10,
-    'Medium': 5,
-    'medium': 5,
-    'Low': 1,
-    'low': 1
-}
 
-loe_dict = {
-    'High': 1,
-    'high': 1,
-    'Medium': 5,
-    'medium': 5,
-    'Low': 10,
-    'low': 10
-}
+def triage_search(auth: Auth):
+    triage_arr = []
+    triage_issues = _search_for_triage(auth)
+    for issue in triage_issues['issues']:
+        print('Retrieving info for issue: {}'.format(issue['key']))
```

Comment:
Debug output?  Can we remove?

Reply:
I actually left that in there on purpose because otherwise there's no indication of what's going on.

Reply:
Ah, OK.  There is no way to do the Pythonic list comprehension with this print statement.  So, this is the way to go, but can you change the `print` statement use `click.echo` function instead.